.github/workflows/generate-ts-declaration.yml

```diff
@@ -41,7 +40,7 @@ jobs:
         uses: actions/checkout@v3
         with:
           repository: aica-technology/frontend
-          ref: develop
+          ref: main
```

Comment:
Interesting, did that workflow never run since we deleted develop on frontend or did we not realized that it fails?

Reply:
this workflow runs only when the schemas change. maybe we haven't modify the schemas since we delete the branch in frontend(?

Reply:
ah i see yeah that's it then probably